src/App.js

```diff
@@ -1,24 +1,21 @@
 import logo from './logo.svg';
 import './App.css';
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
+import HeaderBar from './components/HeaderBar';
+import Register from './components/Register';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <BrowserRouter>
+      <div className="App">
+        <HeaderBar />
+        <div className="page-body">
+          <Switch>
+            <Route path="/register" component={Register}></Route>
```

Comment:
You can keep these route paths in a separate file (see next comment).

example (`routes.js`):

```
export const ROUTES = {
    REGISTER: "/register"
}
```

After that, you can access these by importing `ROUTES`  and using `ROUTES.REGISTER`.